app/views/shared/_ethical_approval_upload.html.erb

```diff
@@ -6,11 +6,13 @@
     <td width="60%"><%= form.check_box :ethically_approved, disabled: !can?(:change_ethically_approved, @study) %></td>
   </tr>
 </table>
+<!--
```

Comment:
If this section is commented out why not delete it?

Reply:
I have deleted the commented code now.